docs/docs-content/release-notes/known-issues.md

```diff
@@ -16,6 +16,7 @@ The following table lists all known issues that are currently active and affecti
 | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ | ---------------------------- |
+| When using content bundles built from cluster profiles using PXK-E as the Kubernetes layer to deploy an Edge cluster, some images in the Kubernetes layer fail be loaded into containerd due to image signature issues, leading to deployment failure.                                                                                                                                                                                                                                                                                                                                    | Remove the `packs.content.images` from the Kubernetes layer in the pack configuration before building the content bundle. These components are already included in the provider image and do not need to be included in the content bundle.                                                                                                                                                                                                                                                                                                                                                                     | December 13, 2024  | Edge                         |
```

Comment:
Lenny, the description feels a bit like a run-on sentence. Maybe we can break it up into two sentences. That would help improve the readability.